dashboard/app/views/levels/_multi.html.haml

```diff
@@ -13,9 +19,10 @@
     %br/
     %br/
 
-    .buttons
-      %a.btn.btn-large.btn-primary.next-stage.submitButton
-        =t('submit')
+    - unless !!(data['options'].try(:[], 'hide_submit'))
```

Comment:
I don't think so, `if`/`unless` will cast `nil` values as `false` already- i'll remove the extra operators